changes/serializers.py

```diff
@@ -11,3 +11,18 @@ class Meta:
         fields = ('id', 'action', 'registrant_id', 'data', 'validated',
                   'source', 'created_at', 'updated_at', 'created_by',
                   'updated_by')
+
+
+class AdminOptoutSerializer(serializers.Serializer):
+    registrant_id = serializers.UUIDField(allow_null=False)
+
+
+class MultipleChangeSerializer(serializers.Serializer):
+    changes = ChangeSerializer(many=True)
+
+    def create(self, validated_data):
+        changes_data = validated_data.pop('changes')
+        changes = []
+        for change_data in changes_data:
+            changes.append(Change.objects.create(**change_data))
+        return changes
```

Comment:
I don't think that we need the `MultipleChangeSerializer`. I think we can just use `ChangeSerializer(many=True)` directly, and pass it a list instead of the dictionary you're passing this serializer